Promote Account Console v3 to preview (#20969)
keycloak/keycloak: docs/documentation/release_notes/topics/22_0_0.adoc

= KeyStore Config Source added

> In relation to the KeyStore Vault news, we also integrated Quarkus's recently released feature called KeyStore Config Source. This means that among the already existing configuration sources (CLI parameters, environment variables and files), you can now configure your Keycloak server via configuration properties stored in a Java keystore file. You can learn more about this feature in the https://www.keycloak.org/server/configuration[Configuration guide].

Reviewer: Hard to see what changed here, but did you mean to change this at all?

Author: It's just a white-space change. It can be safely ignored.

> In relation to the KeyStore Vault news, we also integrated Quarkus's recently released feature called KeyStore Config Source. This means that among the already existing configuration sources (CLI parameters, environment variables and files), you can now configure your Keycloak server via configuration properties stored in a Java keystore file. You can learn more about this feature in the https://www.keycloak.org/server/configuration[Configuration guide].

Reviewer 2: ```suggestion
In relation to the KeyStore Vault news, we also integrated Quarkus' recently released feature _KeyStore Config Source_. Therefore, in addition to the existing configuration sources (CLI parameters, environment variables, and files), you can now configure your Keycloak server by configuration properties stored in a Java keystore file. For more details about this feature, see the https://www.keycloak.org/server/configuration[Configuration guide].
```

= Account Console v3 promoted to preview

> In version 21.1.0 of Keycloak the new Account Console (version 3) was introduced as an experimental feature. Starting this version it has been promoted to a preview feature.

Reviewer 2: ```suggestion
In version 21.1.0 of Keycloak, the new Account Console (version 3) was introduced as an experimental feature. Starting with this version, that console is promoted to a preview feature.
```

Author: Fixed :+1:
